yarn.lock

```diff
@@ -2,11 +2,42 @@
 # yarn lockfile v1
 
 
+"@bazel/bazel-darwin_x64@2.0.0":
```

Comment:
I think the lock file just needs to be regenerated after the last package.json changes